assets/app/views/project.html

```diff
@@ -75,7 +75,8 @@ <h2 class="service" ng-if="!displayRouteByService[serviceId]">
                 <span ng-if="service.spec.portalIP != 'None'">
                   routing traffic on {{service.spec.portalIP}}
                 </span>
-                <span ng-if="numPorts">&#8211;
+                <span ng-if="numPorts">
```

Comment:
It might work, but it's better to just change them to `div` elements if you want them to be block level. Then it's obvious they're block elements when you're looking at the markup.